Makefile

```diff
@@ -79,6 +79,14 @@ go_protoc-go-inject-tag:
 	fi; \
```

Comment:
The only piece this PR is really missing is an easy (gif? copy-pastable curl? make target?) to try out the features introduced.

Reply:
💯, I'll come up with something and update

Reply:
This is gonna be added in #169 since it has both CLI and RPC and it makes sense to have shared stuff in there.

Reply:
I have updated the PR description with a "demo" https://github.com/pokt-network/pocket/pull/169 

This is mostly for your eyes guys, depending on your feedback I will repurpose it accordingly and maybe I'll remove the "jokes" as well. ;)

Reply:
https://github.com/pokt-network/pocket/pull/169#issuecomment-1301548832